components/nexus-blobstore-file/src/main/java/org/sonatype/nexus/blobstore/file/MapdbBlobMetadataStore.java

```diff
@@ -437,6 +444,23 @@ public long getTotalSize() {
   }
 
   @Override
+  public long getBlobSize() {
+    ensureStarted();
+    return database.execute(new Fun.Function1<Long, DB>()
+    {
+      @Override    
+      public Long run(final DB db) {
+        Long totalSize = 0l;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this lower case "l" long suffix by an upper case "L". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALowerCaseLongSuffixCheck)